Dockerfile

```diff
@@ -48,9 +53,15 @@ ENV EASYRSA /usr/share/easy-rsa
 ENV EASYRSA_PKI $OPENVPN/pki
 ENV EASYRSA_VARS_FILE $OPENVPN/vars
 
-VOLUME ["/etc/openvpn"]
+# Defaults for /generate_certs.sh
+ENV S3_REGION=""
```

Comment:
Still flushing out some of the defaults, i'll probably borrow an s3 bucket somewhere to verify the rest.